values.schema.json

```diff
@@ -115,6 +115,50 @@
           "items": { "$ref": "#/definitions/clusterConfig" }
         }
       }
+    },
```

Comment:
Please adjust values.yaml and jingress.yaml and them fix schema accordingly.
Please refer to the specification https://json-schema.org/ for the supported fields and definitions.